### PR TITLE
chore: cleanup and reorganise mixins

### PR DESCRIPTION
**Pull request by dated**
- reorganises the mixins into separate modules
 - removes unused mixins and the respective tests
 - general clean up of unit tests

## What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactor
- [ ] Performance
- [x] Tests
- [ ] Build
- [ ] Documentation
- [ ] Code style update
- [ ] Continuous Integration
- [x] Other, please describe: chore

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Does this PR release a new version?

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch
- [x] All tests are passing
- [x] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Comment by dated** [comment]
Except `The numeric literal '10.123456789123456789' will have at different value at runtime.` the issues reported by Codacy appear to be false positives, or am I missing something @faustbrian?

**Comment by faustbrian** [comment]
Yeah that is false positive, we should probably setup https://github.com/ArkEcosystem/core/blob/master/.codacy.yml for explorer too to ignore the tests directory.

**Comment by dated** [comment]
#676 🤷‍♂

**Comment by dated** [comment]
I think the problem in this specific case is that Codacy thinks that `value` could end up with too much precision. Maybe explicitly parsing value to an integer before the division solves this? i.e. `value = parseInt(value) / 1e8`

**Comment by dated** [comment]
The `readableCurrency` method needs some refactoring in order to guarantee that `value` never ends up with too much precision.

A workaround that might work and trick Codacy for the time being and until bignum support is added: explicitly parse `value` as int and normalise before multiplying by `rate`.

**Comment by dated** [comment]
Looks like Codacy is stuck - can you reanalyse @faustbrian?

File: __tests__/unit/specs/blockchain.spec.js [hunk 1]
```diff
@@ -6,12 +6,12 @@ describe('Blockchain Service', () => {
     store.dispatch('network/setServer', 'https://explorer.ark.io/api/v2')
   })
 
-  it('should return the block height', async () => {
+  it('Should return the block height', async () => {
```
Comment (faustbrian) [inline __tests__/unit/specs/blockchain.spec.js]: Why was the casing of this changed? All our projects follow the full lowercase `it('should`.

Comment (dated) [reply __tests__/unit/specs/blockchain.spec.js]: My bad, it was a mix of lower- and uppercase all around.

**Comment by dated** [comment]
`Current Status: Not Analysed` 🤷‍♂ 

Clicking on `Click to reanalyse` doesn't seem to do anything.

**Comment by faustbrian** [comment]
Contacted support and seems to be resolved. There is 1 issue left and then the PR should be ready to be merged.

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 3. _It will earn you $25 USD._

**Pull request merged**